### PR TITLE
cloudwatch: add queryMode attribute

### PR DESCRIPTION
**Pull request by nicolas-fidel-wmx**
## What does this do?
To generate certain dashboards with the Cloudwatch datasource, specify the queryMode

## Why is it a good idea?
Fix a simple issue

## Context
For many queries I need this option

### CODE REVIEW

File: grafanalib/cloudwatch.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class CloudwatchMetricsTarget(Target):
     :param hide: controls if given metric is displayed on visualization
     :param datasource: Grafana datasource name
```
Comment (JamesGibo) [inline grafanalib/cloudwatch.py:33]: Please could you add a documentation string for this new parameter? 
```suggestion
    :param datasource: Grafana datasource name
    :param queryMode: adnafoinsdfinsoginsoigsfoigsfog
```

**Review by JamesGibo — CHANGES_REQUESTED** [review]
Looks good, would you be able to add some documentation for the new parameter added?
Thanks

File: grafanalib/cloudwatch.py (hunk 1 above)
Comment (nicolas-fidel-wmx) [reply grafanalib/cloudwatch.py:33]: I added  a comment

**Review by JamesGibo — APPROVED** [review]

**Comment by nicolas-fidel-wmx** [comment]
When are you going to publish the release or the fix?

**Pull request merged**